fhir-examples/src/main/resources/json/ibm/basic/BasicToken.json

```diff
@@ -57,15 +54,13 @@
       }
     },
     {
-      "__comment": "If the system is present, and there is no code, then this is understood to mean that there is no suitable code in the system in which to represent the code.",
```

Comment:
are these removed as no longer functional or the details are in the test?

Reply:
in past (DSTU2 timeframe) our parser was lenient by default and ignored unexpected elements.  now its strict by default and so these cause issues.
john tells me there is still "special" support for a field called `fhir_comments` but indicated that he might like to remove that in the future and so I stayed away.